src/components/Filters.js

```diff
@@ -0,0 +1,109 @@
+                break;
+            case 'emboss':
+                filterImage.src = amsterdamEmboss;
+                description.innerHTML = "The <em>emboss</em> filter emphasizes the difference between pixels.";
+                break;
+            case 'left sobel':
+                filterImage.src = amsterdamLeftSobel;
+                description.innerHTML = "The <em>sobel</em> filter detects edges in an image.";
+                break;
+            case 'outline':
+                filterImage.src = amsterdamOutline;
+                description.innerHTML = "The <em>outline</em> filter emphasizes <em>large</em> differences between pixels to detect edges.";
+                break;
+            case 'right sobel':
+                filterImage.src = amsterdamRightSobel;
+                description.innerHTML = "The <em>sobel</em> filter detects edges in an image.";
+                break;
+            case 'sharpen':
+                filterImage.src = amsterdamSharpen;
+                description.innerHTML = "The <em>sharpen</em> filter emphasizes small differences between adjacent pixels.";
+                break;
+            case 'top sobel':
+                filterImage.src = amsterdamTopSobel;
+                description.innerHTML = "The <em>sobel</em> filter detects edges in an image.";
+                break;
+            default:
+                break;
+        }
+    }
+
+    handleChange = event => {
+        this.setState({
+            value: event.target.value
+        }, () => {
+            this.changeImage(this.state.value);
+        });
+    }
+    
+    render() {
+    return (
+        <section className="section blue" id="filters-holder">
+            
+            <img src={amsterdamRegular} alt="black and white amsterdam" width="500px"></img>
+            
+            <form className="card" id="form">
+                <label>
+                    Choose a filter:
+                    <select value={this.state.value} onChange={this.handleChange}>
+                        <option selected value="identity">Identity</option>
+                        <option value="blur">Blur</option>
+                        <option value="bottom sobel">Bottom sobel</option>
+                        <option value="emboss">Emboss</option>
+                        <option value="left sobel">Left sobel</option>
+                        <option value="outline">Outline</option>
+                        <option value="right sobel">Right sobel</option>
+                        <option value="sharpen">Sharpen</option>
+                        <option value="top sobel">Top sobel</option>
+                    </select>
+                </label>
+            </form>
+            
+            <img id="filter-applied" src={amsterdamRegular} alt="amsterdam with filter applied" width="500px"></img>
+            
+            <p className="card" id="img-description">What do you think the identity filter does?</p>
```

Comment:
> In the description below the images, the card component has a column flex orientation, but this is also applied to `<em>` tags within `<p>` tags (when I change them using JS). How can I make it so the `<em>` is displayed in the usual way, in line with its surrounding text?

The source of your strife is right around here. Since you declared the `<p>` tag to be of class `.card`, the `display: flex` and all flexbox properties are applied to its **child nodes**. Notice the phrasing. Since we are applying this flexbox to its child nodes, the `<em>` tags you draw after switching the filter will be treated as children, just like the text around them, and placed into a column. You can solve this problem with something like this:

```suggestion
            <div className='card'>
                <p id='img-description'>What do you think the identity filter does?</p>
            </div>
```